### PR TITLE
Add ability to control error message from a scalar transformer coercion

### PR DESCRIPTION
**Pull request by hlship**
This gives users the option  to  customize the error message when a scalar can't be transformed.  This will be quite valuable for some of the improvements to schemas we've been discussing.

### CODE REVIEW

File: src/com/walmartlabs/lacinia/schema.clj [hunk 1]
```diff
@@ -14,7 +14,7 @@
     [com.walmartlabs.lacinia.internal-utils
      :refer [map-vals map-kvs filter-vals deep-merge q
              is-internal-type-name? sequential-or-set? as-keyword
-             combine-results]]
+             combine-results cond-let]]
```
Comment (annapawlicka) [inline src/com/walmartlabs/lacinia/schema.clj]: `cond-let` isn't used anywhere in this ns

File: src/com/walmartlabs/lacinia/util.clj [hunk 2]
```diff
@@ -25,25 +27,34 @@
            :else
            ;; If resolver-k is not a keyword, it must be a sequence,
            ;; in which first element is a key that points to a resolver
-           ;; factory in resolver-m and subsequent elements are arguments
+           ;; factory in resolver-m and subsequent elements are argumentsc
```
Comment (annapawlicka) [inline src/com/walmartlabs/lacinia/util.clj]: Keybinding gone wrong?

File: src/com/walmartlabs/lacinia/schema.clj (hunk 1 above)
Comment (hlship) [inline src/com/walmartlabs/lacinia/schema.clj]: Well, maybe it should be :-)

**Comment by bcarrell** [comment]
Conflicts

**Comment by hlship** [comment]
Wait, I'm responsible for conflicts while I sleep?  

I'll fix this up when I get to the office.

**Comment by hlship** [comment]
Should be ready to go.

**Pull request merged**